client/tsconfig.json

```diff
@@ -2,7 +2,7 @@
   "compilerOptions": {
     "target": "ES2020",
     "useDefineForClassFields": true,
-    "lib": ["ES2020", "DOM", "DOM.Iterable"],
+    "lib": ["ES2020", "DOM", "DOM.Iterable", "ES2021.String"],
```

Comment:
What does this do?

Reply:
This allows for the use of string.replaceAll() with typescript see error mentioned here https://bobbyhadz.com/blog/typescript-string-replace-all-occurrences

Reply:
ooh thanks will take a read